Content.Server/GuideGenerator/ReagentEntry.cs

```diff
@@ -57,22 +57,28 @@ public sealed class ReactionEntry
     [JsonPropertyName("products")]
     public Dictionary<string, float> Products { get; }
 
-    [JsonPropertyName("effects")]
-    public List<ReagentEffect> Effects { get; }
+    [JsonPropertyName("startEffects")]
+    public List<ReagentEffect> StartEffects { get; }
+    [JsonPropertyName("stepEffects")]
+    public List<ReagentEffect> StepEffects { get; }
+    [JsonPropertyName("stopEffects")]
+    public List<ReagentEffect> StopEffects { get; }
```

Comment:
I need to figure out what this is used for and what I need to change to handle this.

Reply:
This is used to autogenerate entries in https://wiki.spacestation14.io/wiki/Chemistry

Reply:
Fixing this'll be a doozy, I recommend if possible using another mediawiki instance and an export of the chemistry page to adjust the lua script.